### PR TITLE
feat: Resolve blog fetching

### LINKED ISSUE #132: Bug: blog post doesn't show up on landing page
### Describe the bug

The Intro to Open Source blog post is on the site, but is only viewable if you have the url. 

### Steps to reproduce

- got to https://opensauced.pizza/blog/
- notice it's not there. 

Here's the live url: https://opensauced.pizza/blog/open-source-101:-a-beginner's-guide-to-getting-started

### Browsers

Chrome

### Additional context (Is this in dev or production?)

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [X] I agree to follow this project's Contribution Docs

### PR DESCRIPTION
**Pull request by shamimbinnur**
## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Description
1. Attempted to fetch "featuredBlogs" from the backend, but it does not exist in the database. Refactored the code to cease the attempt of fetching it (It was also being fetched in some other parts of the code, requiring an inspection to remove those instances as well).

2. Initially, all blogs were fetched unsorted from the backend, resulting in potential blog omissions when sorting was done on the frontend. This sorting operation shouldn't be performed on the frontend. Refactored the code to fetch sorted data directly from the backend.

## Related Tickets & Documents
This fixes #132 

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

### CODE REVIEW

File: lib/sanity.ts [hunk 1]
```diff
@@ -145,7 +145,7 @@ export const getBlogs: (limit: number) => Promise<SanityBlog[]> = async (
 
 export const getAllBlogs: () => Promise<SanityBlog[]> = async () => {
   const allBlogs: SanityBlog[] = await client.fetch(
-    `*[_type == 'blog' && !(_id in path('drafts.**'))]  {
+    `*[_type == 'blog' && !(_id in path('drafts.**'))] | order(_createdAt desc)  {
```
Comment (bdougie) [inline lib/sanity.ts:148]: What is the context for adding this here? Checking the sanity documentation to educate myself and seeing a different pattern. 

https://www.sanity.io/docs/sort-orders

Comment (shamimbinnur) [reply lib/sanity.ts:148]: @bdougie the doc link you looked at describes how you can display sorted documents in the **Sanity Studio**. But here the `order(_createdAt desc)` is used to **query** the sorted data as an API response.

Please take a look at this URL
https://www.sanity.io/docs/query-cheat-sheet#b5aec96cf56c

Comment (bdougie) [reply lib/sanity.ts:148]: Excellent, thanks for the clarification

**Comment by bdougie** [comment]
Really appreciate the work on this.

**Pull request merged**